onnxruntime/core/optimizer/skip_layer_norm_fusion.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include "core/optimizer/initializer.h"
+#include "core/optimizer/skip_layer_norm_fusion.h"
+#include "core/graph/graph_utils.h"
+#include "float.h"
+#include <deque>
+
+using namespace ONNX_NAMESPACE;
+using namespace onnxruntime::common;
+namespace onnxruntime {
+
+// LayerNorm supports limited data types.
+static std::vector<std::string> supported_data_types{"tensor(float16)", "tensor(float)", "tensor(double)"};
```

Comment:
SkipLayerNormalization does not support double right now.

Reply:
Will it be supported in the future? If it will, we can keep it here?

Reply:
Bert model is usually large (for example, base model can be about 0.5GB). I have not seen model saved in double, which will make model size almost double.

How about removing it right now?  If it is supported in the future, we can add it back easily. Otherwise, we might get a fused node which cannot be run by current binary.